analyzers/tests/SonarAnalyzer.Test/TestCases/UsePascalCaseForNamedPlaceHolders.Latest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Microsoft.Extensions.Logging;
+
+// https://github.com/SonarSource/sonar-dotnet/issues/9545
+public class Repro_9545
+{
+    public void Method(ILogger logger, int number)
+    {
+        logger.LogDebug($@"{nameof(Repro_9545)} filter: {{number}}", number);       // Compliant - FN
+        logger.LogDebug(@$"{nameof(Repro_9545)} filter: {{number}}", number);       // Compliant - FN
+        logger.LogDebug($$"""{{nameof(Repro_9545)}} filter: {number}""", number);   // Compliant - FN
```

Comment:
This is a bit surprising. It actually should raise given that implementation looks into interpolated string text portions. Nevermind. It is documented now.